### PR TITLE
Added option to open Brief in a popup instead of a new tab

### PR DESCRIPTION
**Pull request by katzelad**
Added option to open Brief in a popup instead of a new tab (#328 #339 #433)
Modified global shortcut key to ctrl+shift+f (since ctrl+shift+d does not work)

### CODE REVIEW

**Comment by katzelad** [comment]
@tanriol please take a look
Thanks

File: scripts/utils.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ export function getPluralForm(number, forms) {
 
 
 export async function openBackgroundTab(url) {
-    let tab = await browser.tabs.getCurrent();
+    let [tab] = await browser.tabs.query({active: true, currentWindow: true});
```
Comment (Sxderp) [inline scripts/utils.js]: I'm unfamiliar with this syntax. What's the brackets signify?

Comment (katzelad) [reply scripts/utils.js]: Assigning the first element of the right hand side array directly to `tab` (the same as `let tab = (await browser.tabs.query({active: true, currentWindow: true}))[0];`).

**Comment by katzelad** [comment]
@tanriol why?

**Comment by tanriol** [comment]
Oh. Looks like this was an unintended side effect of the `devel` branch being deleted after merge into `master`. Sorry, I had no idea of this kind of side effects on github. Looks like I'm even unable to reopen the PR because there's no target branch left.

**Comment by katzelad** [comment]
@tanriol Are you still maintaining this extension?

**Comment by tanriol** [comment]
Trying to. The current big item is feed preview / subscription in modern Firefox versions.

**Pull request closed without merging**